tests/e2e/scale/noobaa/test_list_objects.py

```diff
@@ -98,3 +103,51 @@ def test_list_large_number_of_objects(
             f"Files count {files_count} and objects_count {objects_count} does not match. "
             "Check does all objects are listed correctly!!!"
         )
+
+    @bugzilla("2110504")
+    @bugzilla("2141555")
+    @bugzilla("2135782")
+    @skipif_ocs_version("<4.12")
+    @pytest.mark.polarion_id("OCS-4650")
+    def test_list_with_prefix_delimiter(self, bucket_factory, scale_cli_pod, mcg_obj):
```

Comment:
Can we combine this test with test_list_large_number_of_objects, most of the steps looks same. Please check

Reply:
already discussed offline, since these two test cover two different scenarios, combining them wont make difference